spec/forms/update_user_mobile_form_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe UpdateUserMobileForm do
+  let(:user) { build_stubbed(:user, :signed_up) }
+  subject { UpdateUserMobileForm.new(user) }
+
+  it do
+    is_expected.
+      to validate_presence_of(:mobile).
+      with_message(t('errors.messages.improbable_phone'))
+  end
+
+  describe 'mobile validation' do
+    it 'uses the phony_rails gem with country option set to US' do
+      mobile_validator = subject._validators.values.flatten.
+                         detect { |v| v.class == PhonyPlausibleValidator }
+
+      expect(mobile_validator.options).
+        to eq(country_code: 'US', presence: true, message: :improbable_phone)
+    end
+  end
+
+  def format_phone(mobile)
+    mobile.phony_formatted(
+      format: :international, normalize: :US, spaces: ' '
+    )
+  end
+
+  describe 'mobile uniqueness' do
```

Comment:
I thought we removed the unique phone number requirement. Am I mis-reading this test?